parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -348,6 +348,8 @@ ensureArtifactStreaming() {
   systemctl start acr-mirror.service
   sudo /opt/acr/tools/overlaybd/install.sh
   sudo /opt/acr/tools/overlaybd/enable-http-auth.sh
+  sudo /opt/acr/tools/overlaybd/config.sh download.enable false
+  sudo /opt/acr/tools/overlaybd/config.sh cacheConfig.cacheSizeGB 32
```

Comment:
Even if storage space in node < 32 Gb, this would not error out, which is ok since this is the cache size after which there will be garbage collection of partial image layers.

Reply:
This increase is primarily for large ML models